src/manager.js

```diff
@@ -84,13 +85,27 @@ AdManager.prototype.initGoogleTag = function() {
   this.googletag.pubads().addEventListener('slotOnload', adManager.onSlotOnload);
 
   this.initBaseTargeting();
+  if (this.options.amazon_enabled) {
+    this.initAmazonA9();
+  }
 
   this.initialized = true;
 
   this.loadAds();
 };
 
 /**
+ * initializes A9
+ *
+ * @returns undefined
+*/
+AdManager.prototype.initAmazonA9 = function() {
+  amznads.getAds('3076');
```

Comment:
These values are duplicated in a few places, could they live in a common config somewhere? (I'm not familiar with ads setup, and maybe it's simpler as-is)